src/main/java/com/kulik/airbnb/dao/GoogleOAuthClient.java

```diff
@@ -41,18 +53,19 @@ public UserDto getUser(String authorizationCode) throws IOException {
                 .setPersonFields(FIELDS)
                 .execute();
 
-        UserDto userDto =  new UserDto(null,
+        User user =  new User(null,
                 profile.getNames() == null ? null : profile.getNames().get(0).getGivenName(),
                 profile.getBirthdays() == null ? null :
-                        new Date(profile.getBirthdays().get(0).getDate().getYear() - 1900,
+                        new GregorianCalendar(
+                                profile.getBirthdays().get(0).getDate().getYear(),
                                 profile.getBirthdays().get(0).getDate().getMonth() - 1,
                                 profile.getBirthdays().get(0).getDate().getDay()),
-                profile.getGenders() == null ? null : profile.getGenders().get(0).getValue(),
+                profile.getGenders() == null ? "unknown" : profile.getGenders().get(0).getValue(),
                 profile.getPhotos() == null ? null : profile.getPhotos().get(0).getUrl(),
                 profile.getEmailAddresses() == null ? null : profile.getEmailAddresses().get(0).getValue(),
-                null, null, null, null, null);
+                true, null, null, null, null);
```

Comment:
I suggest using setters and default constructor for User (w/o parameters) instead of such a complicated one-statement solution